cfme/common/__init__.py

```diff
@@ -420,6 +420,9 @@ def add_tag(self, tag=None, cancel=False, reset=False, details=True, dashboard=F
                             default is details page
             dashboard (bool): Set to True if tag should be added via the Dashboard view
         """
+        if tag and tag in self.get_tags():
+            logger.warning('Trying to add tag [%s] already assigned to entity [%s]', tag, self)
```

Comment:
why not `format` here?

Reply:
@lina-nikiforova for logger better to use normal format. If you check for conf you will get the complate idea. 

I understand good to use `.format` or `fstring` :+1:

Reply:
@digitronik thanks, it's just a question :) because I briefly checked other logging messages we have in the codebase and I saw that format is used

Reply:
@lina-nikiforova ones that you found that were formatting directly should be updated - thanks for asking/looking!